pom.xml

```diff
@@ -32,11 +33,16 @@
         <maven-plugin-annotations.version>3.13.0</maven-plugin-annotations.version>
         <eclipse-collections-api.version>11.1.0-r11</eclipse-collections-api.version>
         <maven-plugin-plugin.version>3.13.0</maven-plugin-plugin.version>
+        <maven-clean-plugin.version>3.3.1</maven-clean-plugin.version>
+        <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
         <jupiter.version>5.12.0</jupiter.version>
         <!-- origin -->
         <origin.working.directory>${project.build.directory}/origin-sources</origin.working.directory>
-        <source.zip>Pilot-Defined-RxNorm-with-SNCT-classes-2024-04-10-with-custom-annotations</source.zip>
+        <source.version>2024-04-10</source.version>
+        <source.zip>Pilot-Defined-RxNorm-with-SNCT-classes-${source.version}-with-custom-annotations</source.zip>
```

Comment:
The source.zip should exist in the Downloads directory : ${user.home}/Downloads/...